milestone_project_1_original.py

```diff
@@ -0,0 +1,121 @@
+"""
+1. Display a board
+2. User can input 1-9
+3. Check if the user input is taken, else, ask for another number
+4. Add the mark to the board
+5. Check if the user wins or tie, else, next player's turn
+6. Play again?
+"""
+
+board = ['-',' ',' ',' ',' ',' ',' ',' ',' ',' ']
+current_player = "Player_1"
+player_1_mark, player_2_mark = ("O","X")
+
+def display_board(board):

```

Comment:
No need to have board as the parameter in this function since board has already been the global variable.

Reply:
fixed